querybook/webapp/components/DataDoc/DataDocCellControl.tsx

```diff
@@ -268,7 +268,10 @@ const InsertCellButtons: React.FC<{
             key={cellKey}
             onClick={() => handleInsertcell(cellKey)}
             icon="plus"
-            title={titleize(cellKey)}
+            title={ !cellTypes[cellKey].name
```

Comment:
can you change the iteration to 
```
Object.entries(cellTypes).map(([cellKey, cellType]) => (
```
since cellTypes[cellKey] is needed


and can we change title to `title={cellType.name ?? titleize(cellKey)}`